iwyu_output.cc

```diff
@@ -609,6 +609,10 @@ void IwyuFileInfo::ReportIncludeFileUse(const clang::FileEntry* included_file,
   LogSymbolUse("Marked use of include-file", symbol_uses_.back());
 }
 
+void IwyuFileInfo::ReportIncludedFileMacroUse(const FileEntry* included_file) {
```

Comment:
I think it would be nicer to collapse this and `ReportPragmaKeep` into one more general concept, something like `ReportKnownDesiredFile`.